content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,90 @@
+# check-ignore
+
+Check whether any given files or directories are excluded from DVC due to the
+patterns found in [`.dvcignore`](/doc/user-guide/dvcignore).
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        File or directory paths to check (wildcards supported)
```

Comment:
I think wildcard == multiple targets. It's not a special support from DVC. The same applies to all other commands, I guess, in DVC that support file path targets.

Reply:
You're right. The shell will expand the wildcard into multiple arguments, right? So we can just not mention this here.

> I did add a note about it in the description as it's a pretty useful trick for this one command.

Reply:
Removed in #1673